Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -142,24 +127,6 @@ localMessages=/usr/local/kitodo/messages/
 # language.force-default=de
 # Default language: If no Accept-Language Http Request header is present, use the following language:
 language.default=de
-# Resource bundle key for the text to show on the primary screen.
-# - allgemeinesTextDemo: predefined with a text listing the example logins
-#       available after setup
-# - allgemeinesText: predefined with a text about the application
-ApplicationHomepageMsg=allgemeinesTextDemo
-# Resource bundle key for the text to show on the "Impressum" screen.
-# For backgrounds on this, see https://en.wikipedia.org/wiki/Impressum#Webpages
-# - impressumTextDemo: predefined without any specific legal statement,
-#       useful only for test installations
-# Note: No warranty whatsoever is made that any of these statements is
-#       correct or precise.
-ApplicationImpressumMsg=impressumTextDemo
-# Resource bundle key for the text to show on the screen "Technical background"
-# - technischerHintergrundTextDemo: example text
-# - technischerHintergrundText: another example text
-ApplicationTechnicalBackgroundMsg=technischerHintergrundTextDemo
-# Resource bundle key for the website
-ApplicationWebsiteMsg=kitodoWebseite
```

Comment:
`kitodoWebseite`is used inside translation files. If you remove it here, you should remove it in translation files too.